lib/shared/screens/admin/shared/helpers/map-settings-ids.js

```diff
@@ -0,0 +1,3 @@
+import {map, compact} from 'lodash';
+
+export default (options) => compact(map(options, (option) => option.id));
```

Comment:
is there an example data structure you could point me to for  both unlocks and options properties that i can test against